pkg/identity/identity.go

```diff
@@ -30,6 +32,18 @@ type Identity struct {
 	LabelsSHA256 string `json:"labelsSHA256"`
 }
 
+// IPIdentityPair is a pairing of an IP and the security identity to which that
+// IP corresponds.
+//
+// WARNING - STABLE API
+// This structure is written as JSON to the key-value store. Do NOT modify this
+// structure in ways which are not JSON forward compatible.
```

Comment:
Mind adding field tags for the JSON formatting?

Reply:
Sorry about that, done.